### ISSUE TITLE
OC doesn't automatically redirect to the page after setting and unsetting the maintenance mode manually

### ISSUE THREAD

**Issue by jvillafanez** [owncloud/core#15419]
### Steps to reproduce
1. Log in with an user
2. From the OC command line set the maintenance mode (`sudo -u www-data ./occ maintenance:mode --on`)
3. With the logged in user, load another page. You'll get redirected to the maintenance screen
4. Unset the maintenance mode (`sudo -u www-data ./occ maintenance:mode --off`)

### Expected behaviour
The page is eventually refreshed, leaving the maintenance screen (according to what the maintenance screen says)

### Actual behaviour
You don't leave the maintenance screen. You need to refresh the page manually

### Server configuration
**Operating system**: ubuntu 14.04

**Web server:** apache 2.4.7

**Database:** mysql

**PHP version:** 5.5.4

**ownCloud version:** OC 8.0.3RC1

**Updated from an older ownCloud or fresh install:** fresh install

### Client configuration
**Browser:** firefox, chrome

### Notes
I have APC installed, which might be the cause of this issue. However, after refreshing the page manually, it loads the correct page, but it could be because the cache has already expired.

**Comment by jvillafanez** [comment]
It might be easier to change the message in order to avoid a big change

**Comment by nickvergessen** [comment]
```
This ownCloud instance is currently in maintenance mode, which may take a while.

This page will refresh itself when the ownCloud instance is available again.

Contact your system administrator if this message persists or appeared unexpectedly.

Thank you for your patience.
```

The problem is the `This page will refresh itself when the ownCloud instance is available again.` part is from the update (not maintenance only).
Changing the language will not be easy (it breaks all translations), but maybe we can just hide it, when the visitor is not the one who updates.

**Comment by PVince81** [comment]
Yes, let's just hide it.

**Comment by MorrisJobke** [comment]
Why not add a simple refresh every minute?

```
<meta http-equiv="refresh" content="60">
```

Should do the trick.

opinions @owncloud/designers

**Comment by karlitschek** [comment]
@MorrisJobke I think this could do the trick

**Comment by MorrisJobke** [comment]
There is already JS in place that should do that, but it doesn't work.

**Comment by MorrisJobke** [comment]
Fix is here: #15483
